third_party/pyth/p2w_autoattest.py

```diff
@@ -116,9 +116,10 @@
 default_attestation_conditions:
   min_interval_secs: 10
 symbol_groups:
-  - group_name: fast_interval_only
+  - group_name: fast_interval_rate_limited
     conditions:
       min_interval_secs: 1
+      rate_limit_interval_secs: 2
```

Comment:
do we want to have a version of this under `default_attestation_conditions` as well? I think in practice we're going to set this to 1 for most batches, and that would save us some configuration.

Reply:
I think that makes sense, will do